Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentSubgraph.php

```diff
@@ -367,8 +368,8 @@ public function findClosestNode(NodeAggregateId $entryNodeAggregateId, FindClose
 
         $queryBuilderRecursive = $this->createQueryBuilder()
             ->select('pn.*, h.subtreetags, h.parentnodeanchor')
-            ->from('ancestry', 'cn')
-            ->innerJoin('cn', $this->nodeQueryBuilder->tableNames->node(), 'pn', 'pn.relationanchorpoint = cn.parentnodeanchor')
+            ->from('ancestry', 'ch')
+            ->innerJoin('ch', $this->nodeQueryBuilder->tableNames->node(), 'pn', 'pn.relationanchorpoint = ch.parentnodeanchor')
```

Comment:
I'm not sure if this change is correct. Actually we get a node with tags and parentnodeanchor.
From my POV it's rather a node than a hierarchy. BUT ... 

Moreover I don't understand why this needs to be in this PR, as this is about "findAncestorNodeAggregateIds"

(same for the other places)

Reply:
i it was from the review comment here https://github.com/neos/neos-development-collection/pull/5261#discussion_r1813588639 and as i copied the query mostly it had to be also "wrong" in other places ...

Reply:
reverted that line:)